Makefile

```diff
@@ -68,7 +68,7 @@ build-all:
 	GOOS=windows go build -o receptor.exe ./cmd/receptor-cl && \
 	GOOS=darwin go build -o receptor.app ./cmd/receptor-cl && \
 	go build example/*.go && \
-	go build -o receptor --tags no_controlsvc,no_backends,no_services,no_tls_config,no_workceptor,no_cert_auth ./cmd/receptor-cl && \
+	go build -o receptor --tags no_backends,no_services,no_tls_config,no_workceptor,no_cert_auth ./cmd/receptor-cl && \
```

Comment:
@shanemcd can you tell us more about these "no_*" build tags please? 

As part of this refactor we are creating an interface in `controlsvc.go` that is being shared with `interface.go` and with the `no_controlsvc` tag this code would not compile. Just want to make sure these tags are ok to drop.